util/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/AetDictionary.java

```diff
@@ -0,0 +1,100 @@
+package com.google.cloud.healthcare.imaging.dicomadapter;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AetDictionary {
+
+  private static final String ENV_AETS_JSON = "ENV_AETS_JSON";
```

Comment:
We should add documentation to the README for this environment variable. (I believe the CMOVE documentation was in the original PR, but missing here)

Reply:
Right, restored relevant part of readme.